packages/ckeditor5-ui/src/search/searchinfoview.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module ui/search/searchinfoview
+*/
+
+import View from '../view';
+
+/**
+ * A view displaying an information text related to different states of {@link module:ui/search/text/searchtextview~SearchTextView}.
+ *
+ * @internal
+ */
+export default class SearchInfoView extends View {
+	/**
+	 * Controls whether the view is visible.
+	 *
+	 * @observable
+	 * @default false
+	 */
+	declare public isVisible: boolean;
```

Comment:
We figured out the same with @DawidKossowski the other day 🙂 The nasty thing is though, that base `View` class does not provide its own template. And if it suddenly did (with this handy `isVisible` binding), every child view would have to use `View#extendTemplate()` instead of `View#setTempate()` which would be a maaaaaaaaaaaasive BC. Plus, on top of that, `View#extendTempate()` does not support adding new children, so that would be useless anyway.

So long story short, with the current architecture, that would have to be... a utility that creates an observable property and then calls `extendTemplate()` itself with the correct binding? Something akin to `CssTransitionDisablerMixin`. Or maybe some very very late `View#render` listener that `extendsTextmplate()` at the last moment?

Reply:
Ah, I didn't know about the need to change `setTemplate()` to `extendTemplate()`. That's a bummer.

> Or maybe some very very late `View#render` listener that `extendsTextmplate()` at the last moment?

I think we should keep it in mind during refactoring UI. We will see what direction we will take and how many BCs we will introduce. Maybe it will be easier to make a decision then.